src/qibocal/protocols/characterization/utils.py

```diff
@@ -208,22 +250,38 @@ def spectroscopy_plot(data, qubit, fit: Results = None):
             label = "qubit frequency"
             freq = fit.frequency
 
-        if fit.amplitude[qubit] is not None:
+        if data.amplitudes[qubit] is not None:
             fitting_report = table_html(
                 table_dict(
                     qubit,
-                    [label, "amplitude"],
-                    [np.round(freq[qubit], 0), fit.amplitude[qubit]],
+                    [label, "amplitude", "chi2 reduced"],
+                    [
+                        (
+                            freq[qubit] * HZ_TO_GHZ,
+                            fit.error_fit_pars[qubit][1] * HZ_TO_GHZ,
```

Comment:
```suggestion
                            freq[qubit],
                            fit.error_fit_pars[qubit][1],
```
In the table we keep `Hz` instead :)

Reply:
....

I don't see the point of this, generally the plots and the table should share the same units.

Reply:
In all tables we show frequencies in Hz.
Seeing errors in with 3 or 5 decimals is not ideal.